doc/source/whatsnew/v0.24.1.rst

```diff
@@ -15,10 +15,34 @@ Whats New in 0.24.1 (February XX, 2019)
 These are the changes in pandas 0.24.1. See :ref:`release` for a full changelog
 including other versions of pandas.
 
+.. _whatsnew_0241.api:
+
+API Changes
+~~~~~~~~~~~
+
+Changing the ``sort`` parameter for :class:`Index` set operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default ``sort`` value for :meth:`Index.union` has changed from ``True`` to ``None`` (:issue:`24959`).
+The default *behavior*, however, remains the same: the result is sorted, unless
+
+1. ``self`` and ``other`` are identical
+2. ``self`` or ``other`` is empty
+3. ``self`` or ``other`` contain values that can not be compared (a ``RuntimeWarning`` is raised).
+
+This change will allow to preserve ``sort=True`` to mean "always sort" in a future release.
+
+The same change applies to :meth:`Index.difference` and :meth:`Index.symmetric_difference`, which
+would do not sort the result when the values could not be compared.
+
+For :meth:`Index.intersection` the option of ``sort=True`` is also renamed
+to ``sort=None`` (but for :meth:`Index.intersection` it is not the default), as
```

Comment:
There's a lot in this sentence. Oh it also isn't quite right... `sort=True` *was* the default, but we fixed that in my other PR...

So maybe something like



    The `sort` option for :meth:`Index.intersection` has changed in three ways.
    
    1. The default has changed from ``True`` to ``False``, to restore the
       pandas 0.23.4 and earlier behavior
    2. The behavior of ``sort=True`` can now be achieved with ``sort=None``.
       This will sort the result only if the values in ``self`` and ``other``
       are not identical.
    3. The value ``sort=True`` is no longer allowed. A future version of pandas
       will support ``sort=True`` meaning "always sort".